doc/api/stream.md

```diff
@@ -573,6 +573,15 @@ This property contains the number of bytes (or objects) in the queue
 ready to be written. The value provides introspection data regarding
 the status of the `highWaterMark`.
 
+##### `writable.writableNeedDrain`
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Is `true` if buffer is full.
```

Comment:
```suggestion
Is `true` if the buffer is full.
```

Also, this seems to be a good indicator to me to maybe call it `writableBufferFull` in the public API instead?

Reply:
@mclina wdyt

Reply:
`writableBufferFull` is something else, i.e. `writableLength >= writableHighWaterMark`. I should change the description here.